data/managed/log_sources/github_audit/log_source.yml

```
@@ -141,6 +142,7 @@ transform: |2-
   }
 
   .client.geo.country_iso_code = del(.json.actor_location.country_code)
+  .client.ip = del(.json.actor_ip)
 
```

we can also populate .related.ip

```suggestion

if .client.ip != null { 
      .related.ip = push(.related.ip, .client.ip) 
}
```

---

also done